php/class-utils.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Utilities for Cloudinary.
+ *
+ * @package Cloudinary
+ */
+
+namespace Cloudinary;
+
+class Utils {
```

Comment:
@dugajean @pereirinha - As mentioned below, we have methods that already are able to do this, so this Class is not needed.

Reply:
The class is alright imo, it should stay there as these utils can come in handy at any time (even if not with this PR).

Reply:
@dugajean I agree that a utils library is useful, but there's no point in introducing code in a PR that is not part of the PR.
When we start refactoring in other places that does warrant a utils ( to reduce duplication) then that is the time to introduce it.